packages/xs-extra/xapi-forkexecd.master/opam

```diff
@@ -9,15 +9,13 @@ build: [
   [make]
 ]
 install: [
-  [make "install"
-		"BINDIR=%{bin}%"
-		"SBINDIR=%{bin}%"
-		"ETCDIR=%{prefix}%/etc"]
+  [make "install"]
+]
+remove: [
+    ["oasis" "setup"]
+    [make "uninstall"]
+    ["ocamlfind" "remove" "forkexec"]
```

Comment:
Maybe OASIS already does this for us? But it probably doesn't hurt to have this repeated here.